h/static/scripts/annotation-ui.js

```diff
@@ -1,8 +1,11 @@
 'use strict';
 
+var immutable = require('seamless-immutable');
+var redux = require('redux');
+
 function value(selection) {
```

Comment:
This seems like an oddly named function to me. Perhaps calling this `freeze` would make its purpose clearer when used later in this file?